apps/web/src/server/applications/case/representations/representation/check-answers/check-answers.view-model.js

```diff
@@ -8,22 +8,23 @@ import { getSelectedOptionsText } from '../utils/get-selected-options-text.js';
  */
 
 /**
- * @param {string} caseId
+ * @param {object|*} query
+ * @param {object|*} params
  * @param {Locals} locals
  * @returns {string}
  */
-const getBackLinkUrl = (caseId, { pageLinks, status }) =>
-	status === 'DRAFT' ? pageLinks.backLinkUrl : getRepresentationBaseUrl(caseId);
+const getBackLinkUrl = ({ repMode }, { caseId }, { pageLinks }) =>
```

Comment:
why destructure and not pass the arg in ? 

Simplify the smaller functions where possible. 

This pattern works well for the big mappers that might need to be easily extended.